java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -596,10 +618,10 @@ long getStartStringOffset(long index) {
   }
 
   /**
-   * Get the ending byte offset for the string at index.
+   * Get the ending byte offset for the nested type or string at index.
    */
   long getEndStringOffset(long index) {
```

Comment:
Can we deprecate this API and have it call into a new one that is named more correctly?, same with getStartStringOffset?

Or revert the comment change above.  We need the assert and the comment to be consistent, and the functions between each other to be consistent.

Reply:
I am going to revert the comment change as it is used for strings only.